packages/api/src/graphql/mutation.public.ts

```diff
@@ -261,6 +273,8 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
       args: {
         email: {type: GraphQLNonNull(GraphQLString)}
       },
+      description:
+        "This mutation allows to reset the password by sending the user's email and a login link will be sent to that email.",
```

Comment:
Maybe better worded:  "This mutation allows to reset the password by accepting the user's email and sending a login to that email.",